store/views.py

```diff
@@ -4,21 +4,27 @@
 from django.contrib.auth.decorators import login_required
 from django.http import JsonResponse
 from django.views.decorators.csrf import csrf_exempt
-
+import datetime
 # Create your views here.
 
 def index(request):
     return render(request, 'store/index.html')
 
 def bookDetailView(request, bid):
-    template_name = 'store/book_detail.html'
     context = {
         'book': None, # set this to an instance of the required book
         'num_available': None, # set this to the number of copies of the book available, or 0 if the book isn't available
+        'rating': '-1',
     }
+    template_name = 'store/book_detail.html'
+    context['book'] = Book.objects.get(id=bid)
```

Comment:
This may fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error.